src/application/application_test.ts

```diff
@@ -46,14 +46,14 @@ test('Application should expose gRPC api', (t) => {
 
 const event: Event = {
     serviceID: 'id',
-    filter: '*',
-    dataFilter: (_, eventData) => eventData['foo'] == 'bar'
+    eventKey: '*',
+    filter: (_, eventData) => eventData['foo'] === 'bar'
```

Comment:
@antho1404 since we're using a transpiler to convert ts code to ecma5, it already deals with _==_ to _===_

Reply:
good point, it's just a replace from the merge, done